tools/releaseTools.psm1

```diff
@@ -341,9 +341,34 @@ function Get-ChangeLog
     PrintChangeLog -clSection $clDocs -sectionTitle 'Documentation and Help Content'
 }
 
+function Get-ChangeLogEx
```

Comment:
Can we build this functionality into `Get-ChangeLog` . (feel free to rename the existing function)

Reply:
Yes we can, this will change the function parameters.

Reply:
I have made new `$ThisReleaseTag` parameter optional.